code/src/components/SeventhQuestionRecommend.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
yes that could be done and might be a more better approach. However, i was too focused to have every thing as a component therefore i went for the separate question component instead of doing it in one component.